dps-gradle-spring-boot-suppressions.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Did you mean to commit this file?  I was expecting it to be added to `.gitignore` and then have a
```
dependencyCheck {
  suppressionFiles = listOf()
}
```
block

Reply:
Yeah probably best - I'll get rid.